misc/package/Data Files/MWSE/core/lib/event.lua

```diff
@@ -93,10 +97,14 @@ function this.register(eventType, callback, options)
 	-- If 'doOnce' was set, wrap with a call to unregister.
 	if options.doOnce then
 		local originalCallback = callback
-		callback = function(e)
-			this.unregister(eventType, callback, options)
+		local function newCallback(e)
+			-- `event.unregister` will handle the conversion for us.
+			event.unregister(eventType, originalCallback, options)
```

Comment:
Let's just call this `this.unregister` to stay consistent with the rest of the file.

Reply:
Good catch. I fixed this in the latest commit.